src/librustc_mir/borrow_check/error_reporting.rs

```diff
@@ -538,6 +574,66 @@ impl<'cx, 'gcx, 'tcx> MirBorrowckCtxt<'cx, 'gcx, 'tcx> {
         err
     }
 
+    fn find_name_and_ty_of_values(
```

Comment:
Can we add a doc comment to this method?

Reply:
This method doesn't exist in recent commits.